js/views/pageNavVw.js

```diff
@@ -350,18 +350,19 @@ module.exports = Backbone.View.extend({
     $('#obContainer').addClass('blur');
 
     // display the modal
-    $('.js-aboutModal').removeClass('hide');
+    $('.js-aboutModal').removeClass('fadeOut').addClass('fadeIn');
   },
```

Comment:
I was trying to get the modal to fadeIn like the transaction modal, but didn't have much luck.